src/config/prediction_settings.py

```
@@ -2,20 +2,21 @@
 ロジスティック方程式の予測設定
 """
 
+
 class PredictionSettings:
     """予測設定を管理するクラス（年単位固定）"""
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)